src/libcore/tests/slice.rs

```diff
@@ -1130,6 +1130,44 @@ fn test_rotate_right() {
     }
 }
 
+#[test]
+#[cfg(not(miri))]
```

Comment:
These should come with a comment explaining why. I will add a `Miri is too slow` in one of the Miri-adjustment PRs I have in flight anyway.